src/modules/users/New/index.js

```diff
@@ -1,23 +1,44 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import queryString from 'query-string'
 
 import Configure from './Configure'
 import Module from '../../app/Module'
 
 class New extends React.Component {
-  title = 'Invite to Workspace'
-  menu = {
-    home: '/users',
+  // state for entire module
+  state = {
+    title: 'Invite to Workspace',
+    menu: {
+      home: '/users',
+    },
+    superuser: false,
+  }
+
+  componentWillMount() {
+    const values = queryString.parse(this.props.location.search) // eslint-disable-line
```

Comment:
We could use the built-in URLSearchParams here, with the https://github.com/ungap/url-search-params for IE support

```suggestion
    const values = new URLSearchParams(this.props.location.search) // eslint-disable-line
```

Reply:
(Or https://github.com/jerrybendy/url-search-params-polyfill - there is, naturally, more than one polyfill optiuon)

Reply:
You're right. I'll update this.